### PR TITLE
quick fix for decrease quantity more than available quantity

### PR DESCRIPTION
**Pull request by Sharptail**

### CODE REVIEW

**Review by creativequotient — APPROVED** [review]
LGTM

**Review by jamesleeht — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/logic/commands/inventory/InventoryCommandUtil.java [hunk 1]
```diff
@@ -7,6 +7,9 @@
 public class InventoryCommandUtil {
     public static final String MESSAGE_INVALID_QUANTITY = "The quantity entered is invalid, "
             + "please enter a positive number only!";
+    public static final String MESSAGE_DECREASE_QUANTITY = "Invalid quantity entered! "
+            + "The quantity to decrease is more than available quantity.";
```
Comment (guanyz) [inline src/main/java/seedu/address/logic/commands/inventory/InventoryCommandUtil.java]: Grammatically, this should be "more than the available quantity" I think!

**Review by IanCKW — APPROVED** [review]
LGTM

**Pull request merged**